Makefile

```diff
@@ -66,11 +66,12 @@ shellcheck: ## Lints Bash and sh scripts.
 # don't maintain those scripts. Omitting the `.venv/` dir because we don't control
 # files in there. Omitting the ossec packages because there are a LOT of violations,
 # and we have a separate issue dedicated to cleaning those up.
-	@find "$(PWD)" \( -path "$(PWD)/.venv" -o -path "$(PWD)/install_files/ossec-server" \
-		-o -path "$(PWD)/install_files/ossec-agent" \) -prune \
+	@find \( -path "./.venv" -o -path "./install_files/ossec-server" \
+		-o -path "./install_files/ossec-agent" \) -prune \
 		-o -type f -and -not -ipath '*/.git/*' -exec file --mime {} + \
 		| perl -F: -lanE '$$F[1] =~ /x-shellscript/ and say $$F[0]' \
-		| xargs shellcheck -x --exclude=SC2001,SC2064,SC2181
+		| xargs docker run -v "$(PWD):/mnt" -t koalaman/shellcheck:v0.4.6 \
+		-x --exclude=SC1091,SC2001,SC2064,SC2181
```

Comment:
Not to be the "hey this didn't work on macOS" person like usual, but... hey this didn't work on macOS (after installing shellcheck via homebrew):

```
$ make shellcheck
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
```

However, if I make the following change:

```
diff --git a/Makefile b/Makefile
index cd74e8b..ad13a9e 100644
--- a/Makefile
+++ b/Makefile
@@ -66,7 +66,7 @@ shellcheck: ## Lints Bash and sh scripts.
 # don't maintain those scripts. Omitting the `.venv/` dir because we don't control
 # files in there. Omitting the ossec packages because there are a LOT of violations,
 # and we have a separate issue dedicated to cleaning those up.
-       @find \( -path "./.venv" -o -path "./install_files/ossec-server" \
+       @find "." \( -path "./.venv" -o -path "./install_files/ossec-server" \
                -o -path "./install_files/ossec-agent" \) -prune \
                -o -type f -and -not -ipath '*/.git/*' -exec file --mime {} + \
                | perl -F: -lanE '$$F[1] =~ /x-shellscript/ and say $$F[0]' \
```

then it works and everything is :100:.

Reply:
Sure, I can make that change—thanks for the patch! By the way, you _don't_ need to install shellcheck via homebrew—it runs entirely outside the container. (`find`, however, runs on your localhost, thus the problems.)